src/Structure/Discord/Client.js

```diff
@@ -135,7 +135,7 @@ class Client {
         }
         this.createMessage(config.discord.edit_log, ':pencil2: | ' +
             (!newFeature ? 'Feature **__' + oldFeature.name + '__** has been deleted.' : (!oldFeature ? 'A feature has been added.' : 'Feature **__' + newFeature.name + '__** has been edited') +
-            '\n<' + config.baseURL + '/lists/features/' + newFeature.id + '>\n\n**Changes**:\n' +
+            '\n<' + config.baseURL + '/lists/features/' + (newFeature.id ? newFeature.id : oldFeature.id) + '>\n\n**Changes**:\n' +
```

Comment:
If you created a new feature, the first in the DB:

`newFeature.id` would be `0`
So it'd fall through to `oldFeature.id`
`oldFeature.id` would be `undefined`

This should be an explicit undefined check: `newFeature.id !== undefined ? newFeature.id : oldFeature.id`